src/RowChart.jsx

```diff
@@ -106,7 +106,7 @@ export class RowChart extends React.Component {
 
   render() {
     return (
-      <div>
+      <div class="product-section">
```

Comment:
this is a generic row chart, so it could be either issue or product, or anything else.

also in the component syntax this should be className
I'd change this to:
className="row-chart-section"